packages/utils/test/toErrorList.test.ts

```diff
@@ -0,0 +1,11 @@
+import { toErrorList } from '../src';
+import { TEST_ERROR_LIST, TEST_ERROR_SCHEMA } from './testUtils/testData';
+
+describe('toErrorList()', () => {
+  it('returns empty array when nothing is passed', () => {
+    expect(toErrorList()).toEqual([]);
+  });
+  it('', () => {
```

Comment:
Should have a test name even if it's basic "validate examples"

Reply:
Oops! Good catch